rest/server_context.go

```diff
@@ -843,18 +844,22 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(ctx context.Context, config
 		return nil, err
 	}
 
-	ctx = base.DatabaseLogCtx(ctx, dbName, contextOptions.LoggingConfig)
-
-	// If audit logging is enabled for the node, log the db setting.  This may not represent a change in
-	// auditing since the last time the db was running on this node, but we don't have any insight into the
-	// previous state.
-	if base.IsAuditEnabled() {
-		if contextOptions.LoggingConfig.DbAuditEnabled() {
-			base.Audit(ctx, base.AuditIDAuditEnabled, base.AuditFields{base.AuditFieldAuditScope: "db", base.AuditFieldDatabase: dbName})
-		} else {
-			base.Audit(ctx, base.AuditIDAuditDisabled, base.AuditFields{base.AuditFieldAuditScope: "db", base.AuditFieldDatabase: dbName})
+	/*
```

Comment:
This code is now commented out, which doesn't seem right.

Reply:
As discussed, removed this as it's being replaced by the db logging on config change.